src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
+@import "./utils/resets";
+@import "./utils/variables";
+@import "./utils/functions";
+
 body {
+  line-height: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
   margin: 0;
 }
+
+.calendar {
+  $block: &;
+
+  display: flex;
+  margin: 150px auto;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: width-calculate($day-size, $gap, $border-size);
```

Comment:
The wrong calculation for this property. Look screenshot.

![Screenshot 2023-02-07 at 17 21 40](https://user-images.githubusercontent.com/54940595/217288060-47a59808-35e9-4ad9-ad5d-939f86f9e80b.png)